opensearch_py_ml/sentence_transformer_model/__init__.py

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+from .sentencetransformermodel import SentenceTransformerModel
```

Comment:
can you try: `from opensearch_py_ml.sentence_transformer_model.sentencetransformermodel import SentenceTransformerModel`

Reply:
I had this from opensearch_py_ml.sentence_transformer_model in the earlier commit, but it will show an integrated test message "circular import error". I think this init is within opensearch_py_ml and within opensearch_py_ml module, so we just need to import from `sentencetransformermodel.py` file and get `SentenceTransformerModel class`